src/js/profiling/integration.ts

```diff
@@ -172,16 +195,16 @@ export const hermesProfilingIntegration: IntegrationFn = () => {
   };
 
   const _createProfileEventFor = (profiledTransaction: Event): ProfileEvent | null => {
-    const profile_id = profiledTransaction?.contexts?.['profile']?.['profile_id'];
+    const profile_id = profiledTransaction?.contexts?.['trace']?.['data']?.['profile_id'];
```

Comment:
NIT: We should have access to trace and data so we could refer to it on the following way:
```suggestion
    const profile_id = profiledTransaction?.contexts?.trace?.data?.['profile_id'];
```